### DISCUSSION TITLE
希望添加pwa网站特性

### DISCUSSION THREAD

**Discussion by SDCOM-0415** [share121/inter-knot#243] [其他]
如题，希望能使用[pwa](https://www.bing.com/search?q=pwa%E5%BA%94%E7%94%A8 "pwa")的网站形式，推荐GitHub页面也使用单独的网站页面形式，类似于[弹出式登录框](https://gitcode.net/SDCOM_0415/img/-/raw/master/pictures/2024/08/9/22_50_26_202408092250588.png "弹出式登录框")，即用即关，希望作者能看到
![image](https://github.com/user-attachments/assets/a28d7bc2-54f0-4c2c-883f-895f6c44d1af)

**Comment by KinoZR** [comment]
确实，需要几个大佬运维
